src/site/_includes/components/Meta.js

```diff
@@ -130,6 +135,31 @@ module.exports = (locale, page, collections, renderData = {}) => {
     `;
   }
 
+  function renderHreflangMeta() {
+    if (!pageUrl) {
+      return;
+    }
+    const url = pageUrl.startsWith('/i18n/')
+      ? pageUrl.replace(i18nRegex, '')
+      : pageUrl;
+
+    // Find i18n equivalents of the current url and heck if they exist.
+    const langhrefs = i18nLocales
+      .map((locale) => [locale, `/i18n/${locale}${url}`])
```

Comment:
should this line use `path.join('i18n', locale, url)` to make it a little more resilient? Just in case `locale` or `url` ever changes and no longer has a slash